avm/src/main.rs

```diff
@@ -167,10 +170,17 @@ fn main() {
             let agz_file = matches.value_of("AGZ_FILE").unwrap();
             let app_name = matches.value_of("app-name").map(String::from);
             let config_name = matches.value_of("config-name").map(String::from);
+            let files = matches.value_of("files");
+            let mut files_b64 = HashMap::new();
+            if let Some(files) = files {
+              let names = files.split(",");
+              for name in names {
+                files_b64.insert(name.to_string(), get_agz_file_b64(name.to_string()).await);
```

Comment:
Are you sure the `get_agz_file_b64` fn works for this?

Reply:
Yep! That's what my second screenshot was about: to show that it decoded just fine.

The name of that function isn't great, but it works perfectly.